refinery/ui/source/js/file-browser/services/factory.js

```diff
@@ -64,9 +64,21 @@
     * Method Definitions
     * ----------------------
     */
+
+    // helper method which returns  whether a cell is editable
+    function isCellEditable (attributeType) {
+      if (!dataSetPropsService.dataSet.is_owner) {
+        return false;
+      }
+      if (attributeType === 'Factors' || attributeType === 'Characteristics') {
+        return true;
+      }
+      return false;
+    }
```

Comment:
I'd prefer a single boolean expression...
```
return dataSetPropsService.dataSet.is_owner && ( attributeType === 'Factors' || attributeType === 'Characteristics' )
```
or
```
return dataSetPropsService.dataSet.is_owner && ['Factors', 'Characteristics'].includes(attributeType)
```
... but this is a matter of taste.